_rules/SC2-4-2-page-has-title.md

```diff
@@ -166,11 +174,21 @@ Empty first `title`.
 </html>
 ```
 
+#### Failed example 5
+
+The `title` only contains a seperator character.
```

Comment:
typo: seperator -> separator

Above there is another typo. On passed example 2 "This page give" -> "This page gives"